css/main.css

```diff
@@ -0,0 +1,221 @@
+body  {
+    background-color: white;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    margin: 20px;
+    color: rgb(46, 44, 44);
+    line-height: 1.8;
+    height: 100%;
+}
+
+#menu  {
+    background-color:  rgb(247, 246, 236);
+    overflow: hidden;
+    position: sticky;
+    top: 0px;
+    transition: 0.3s;
+    height: 60px;
+    z-index: 1;
+    letter-spacing: 5px;
+   
+}
+@media screen and (max-height: 300px)  {
+#navbar-right a{
+  float: left;
+     }
+}
+#menu a {
+    border-radius: 20px;
+    color: rgb(46, 44, 44);
+    text-decoration: none;
+    padding: 8px;
+    font-weight: bold;  
+    line-height: 35px;
+}
+#menu #fontlogo  {
+    transition: 0.3s;
+    font-weight: bold;
+}
+
+#menu a:hover  {
+    background-color: rgb(207, 207, 207);
+    transition: 0.5s;
+    
+}
+#main-image  {
+    background: url(../picture/oil.jpg) no-repeat fixed center;
+    height: 400px;
+    width: 100%;
+    background-size: cover; 
+    filter: grayscale(20%);
+}
+
+.text  {
+    text-align: center;
+    color:  rgb(247, 241, 241);
+    position: absolute;
+    top: 50%;
+    left:40%;
+}
+.introduction {
```

Comment:
for what is this introduction ? This class name is unclear for me. Maybe you should specify this name by location in html section or by some specific attributes which this class contains

Reply:
Ok, but i have to change all names of classes. It's only first draft version.

Reply:
Ok, so I will waiting for your changes.

Reply:
So i think it will be a first step of my changes. Love.